### ISSUE TITLE
Incompatibility with Tinkers

### ISSUE THREAD

**Issue by TimGoll** [SirLyle/Lanolin#3]
Your mod isn't compatible with Tinkers and its addons

Tinkers' Tools default to minecraft gold tools:
![image](https://user-images.githubusercontent.com/13639408/46548437-5da9e180-c8cf-11e8-836c-e7e20ace02d3.png)

Used tinkers tools are not upgradeable
![image](https://user-images.githubusercontent.com/13639408/46548454-6ac6d080-c8cf-11e8-8362-77572b89897b.png)

Tinkers Armory armor does work
![image](https://user-images.githubusercontent.com/13639408/46548484-7adeb000-c8cf-11e8-9184-b7596677f862.png)

but [crashes the game](https://gist.github.com/TimGoll/8c2aa2fc38ef2c4a441b357614aebbf6). But this might be caused by Tinkered Hegemony by @asiekierka 
maybe he can check this error and fix the crash on his side

Forge: forge-1.12.2-14.23.4.2760
TH: TinkeredHegemony-0.2.0
TC: TConstruct-1.12.2-2.10.1.87
Lanolin: lanolin-1.12-0.2b

**Comment by asiekierka** [comment]
Nah, it's probably that Tinkered Hegemony patches Lanolin's recipes for vanilla pickaxes. This is most likely a TH bug, all of it.

**Comment by TimGoll** [comment]
Ah great. I knew it was a good idea to mention you here :D 

Edit: If it'd help I can test this later

**Comment by asiekierka** [comment]
Throw out TH. It will probably fix all of these issues.

If so, re-report on the TH GitHub.

**Comment by TimGoll** [comment]
@SirLyle Without TH your mod does not detect the tinkers tools at all
@asiekierka armor pieces do work fine without TH. I'll open an issue

**Comment by SirLyle** [comment]
You managed to report this right as I am about to be gone for a week. I was honestly unaware anyone used this mod.

The lack of tinker's compat is likely because they do not extend the vanilla tool classes. I'm gonna mark this as an enhancement request on my end and will look at extending lanolin to tinkers.

**Comment by TimGoll** [comment]
ah no problem! I'm currently creating the pack and it needs some time until it is finished. And I quite like this mod
